frontend/src/metabase-lib/urls.ts

```diff
@@ -95,55 +89,35 @@ export function getUrlWithParameters(
   });
 }
 
-export function getAutomaticDashboardUrl(
-  question: Question,
-  filters: (RawFilter | Filter[])[],
-) {
-  let cellQuery = "";
-
-  if (filters.length > 0) {
-    const mbqlFilter = filters.length > 1 ? ["and", ...filters] : filters[0];
-    cellQuery = `/cell/${utf8_to_b64url(JSON.stringify(mbqlFilter))}`;
-  }
-
+export function getAutomaticDashboardUrl(question: Question) {
```

Comment:
Make the function works with `Question` only. Do not use any deprecated `query()` calls